### ISSUE TITLE
Class Drupal\bootstrap\Bootstrap not found

### ISSUE THREAD

**Issue by tormi** [contentacms/contenta_jsonapi#31]
Tried quick install and failed:

```
Error: Class &#039;Drupal\bootstrap\Bootstrap&#039; not found in include_once() (line 30 of /web/themes/contrib/bootstrap/bootstrap.theme).
Drush command terminated abnormally due to an unrecoverable error.   [error]
 ERROR  The Drupal installer failed to install Contenta CMS.
Script scripts/development/build-contenta_jsonapi.sh handling the install-contenta event returned with error code 3
```

**Comment by e0ipso** [comment]
Yeah. I broke the installer. I'll revert the last commit soon.

**Comment by e0ipso** [comment]
@tormi that should have been fixed by 0ba5212f95b84f1ee77cbbf54f5864761f5db9dc.

**Comment by tormi** [comment]
Thanks! Bootstrap error is gone, but now this: #33 ;)
